### DISCUSSION TITLE
Track appearance will be reset for all tracks by the Track dialog

### DISCUSSION THREAD

**Discussion by rrkommer** [osmandapp/OsmAnd#11114] [Q&A]
Hi, 
I’m using osmand for planning hiking on a tablet (kudos to the new planning tool) and also recording trips on a smartphone.
On the tablet I want have 2 different colors for tracks, where I may want to go and where I already was. 
I menu Configure Map / Tracks I manage the recorded and planned Tracks (gpx files).
In the map itself I can click on a track and change its appearance, like color for each individual track.
Now my problem is, every time I open Configure Map / Tracks dialog and click OK (even don’t change anything) all track styes are reset to the appearance, which can be edited at the top of the Tracks dialog. That makes the possibility to change the appearance of individual tracks virtual useless.
I would prefer, that if the Tracks dialog is closed only tracks, which has no individual style, is set to appearance from the Track dialog.
Is there any way to manage Tracks without this side effect?
One workaround showing tracks on two different maps – f.e. the Globus Map and the Hiking Map – is also not working, because - despite the Tracks menu is on the map specific menu - changes in the track dialog effects all maps.
Any other tips?

Thanks for the great appliation.

**Comment by vshcherb** [comment]
Yes, it's very tricky today with that dialog. We want to make it less confusing whether it affects 1 or many tracks, so this dialog is better to use only when you want to change multiple tracks. And you can click on the map and select Track -> Open Track and change for single track.
